pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -18,60 +52,28 @@ class TestSparseArrayArithmetics:
     def _assert(self, a, b):
         tm.assert_numpy_array_equal(a, b)
 
-    def _check_numeric_ops(self, a, b, a_dense, b_dense):
+    def _check_numeric_ops(self, a, b, a_dense, b_dense, mix, op):
         with np.errstate(invalid="ignore", divide="ignore"):
             # Unfortunately, trying to wrap the computation of each expected
             # value is with np.errstate() is too tedious.
 
-            # sparse & sparse
-            self._assert((a + b).to_dense(), a_dense + b_dense)
-            self._assert((b + a).to_dense(), b_dense + a_dense)
-
-            self._assert((a - b).to_dense(), a_dense - b_dense)
-            self._assert((b - a).to_dense(), b_dense - a_dense)
-
-            self._assert((a * b).to_dense(), a_dense * b_dense)
-            self._assert((b * a).to_dense(), b_dense * a_dense)
-
-            # pandas uses future division
-            self._assert((a / b).to_dense(), a_dense * 1.0 / b_dense)
-            self._assert((b / a).to_dense(), b_dense * 1.0 / a_dense)
-
-            # ToDo: FIXME in GH 13843
-            if not (self._base == pd.Series and a.dtype.subtype == np.dtype("int64")):
-                self._assert((a // b).to_dense(), a_dense // b_dense)
-                self._assert((b // a).to_dense(), b_dense // a_dense)
-
-            self._assert((a % b).to_dense(), a_dense % b_dense)
-            self._assert((b % a).to_dense(), b_dense % a_dense)
-
-            self._assert((a ** b).to_dense(), a_dense ** b_dense)
-            self._assert((b ** a).to_dense(), b_dense ** a_dense)
-
-            # sparse & dense
-            self._assert((a + b_dense).to_dense(), a_dense + b_dense)
-            self._assert((b_dense + a).to_dense(), b_dense + a_dense)
-
-            self._assert((a - b_dense).to_dense(), a_dense - b_dense)
-            self._assert((b_dense - a).to_dense(), b_dense - a_dense)
-
-            self._assert((a * b_dense).to_dense(), a_dense * b_dense)
-            self._assert((b_dense * a).to_dense(), b_dense * a_dense)
-
-            # pandas uses future division
-            self._assert((a / b_dense).to_dense(), a_dense * 1.0 / b_dense)
-            self._assert((b_dense / a).to_dense(), b_dense * 1.0 / a_dense)
+            if op in [operator.floordiv, ops.rfloordiv]:
+                # FIXME: GH#13843
+                if self._base == pd.Series and a.dtype.subtype == np.dtype("int64"):
```

Comment:
can you exfail the ones that are not defined / working

Reply:
done; green